set-web/src/main/java/eu/europeana/set/web/service/controller/BaseRest.java

```diff
@@ -202,72 +140,28 @@ protected String serializeUserSet(LdProfiles profile, UserSet storedUserSet) thr
 		return serializedUserSetJsonLdStr;
 	}
 
-	/**
-	 * This method checks timestamp if provided within the "If-Match" HTTP
-	 * header, if false responds with HTTP 412
-	 * 
-	 * @param request
-	 * @param modified
-	 * @throws ApplicationAuthenticationException
-	 */
-	public void checkHeaderTimestamp(HttpServletRequest request, UserSet userSet)
-			throws ApplicationAuthenticationException {
-		int modified = userSet.getModified().hashCode();
+    /**
+     * This method compares If-Match header with the current etag value.
+     * 
+     * @param etag    The current etag value
+     * @param request The request containing If-Match header
+     * @throws HttpException
+     */
+    public void checkIfMatchHeader(int etag, HttpServletRequest request) throws HttpException {
```

Comment:
this method is now implemented in API Commons, please remove it

Reply:
This method is already not in BaseRest